app/eth2wrap/eth2wrap.go

```diff
@@ -165,14 +166,26 @@ func submit(ctx context.Context, clients []Client, work func(context.Context, Cl
 	return err
 }
 
-// latency measures endpoint latency.
+// latency measures endpoint latency and writes metrics and logs results.
 // Usage:
 //
 //	defer latency("endpoint")()
-func latency(endpoint string) func() {
+func latency(ctx context.Context, endpoint string, enableLogs bool) func() {
+	if enableLogs {
+		log.Debug(ctx, "Calling beacon node endpoint...", z.Str("endpoint", endpoint))
+	}
 	t0 := time.Now()
+
 	return func() {
-		latencyHist.WithLabelValues(endpoint).Observe(time.Since(t0).Seconds())
+		rttSeconds := time.Since(t0).Seconds()
+		latencyHist.WithLabelValues(endpoint).Observe(rttSeconds)
+		if enableLogs {
+			log.Debug(ctx, "Beacon node call finished", z.Str("endpoint", endpoint))
+			// If BN call took more than 1 second, send WARN log
+			if rttSeconds > 1 {
+				log.Warn(ctx, "Beacon node call took longer than expected", nil, z.Str("endpoint", endpoint), z.F64("rtt_seconds", rttSeconds))
```

Comment:
`time.Duration` has nice String() function: https://pkg.go.dev/time#example-Duration.String
Better to print that instead of seconds?

Reply:
Yeah, might be better